.travis.yml

```diff
@@ -4,46 +4,42 @@ sudo: false
 cache:
   apt: true
   directories:
-    - $HOME/.pip-cache/
+    - $HOME/.cache/pip
+
+addons:
+  postgresql: "9.6"
+  apt:
+    packages:
+      - expect-dev
```

Comment:
All these packages are not used anymore since that we are using pip wheels with backup of cache directory

Reply:
@moylop260  
Done. I left apt dependencies as they are, without changes.

Reply:
@moylop260 
You removed all packages, but I think those were necessary. Travis is failing now (see #1300's log [here](https://travis-ci.org/Vauxoo/addons-vauxoo/jobs/445457431#L958)).

It's failing to run unbuffer because the package  `expect-dev` is not installed.

Reply:
You are right.
Let’s add that package again